docs/cloud/shared-infrastructure/ops/README.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Clusters
+dir:
+  text: "Operations"
+  order: 3
+---
+
+## Viewing clusters
+
+When you create a cluster, from the Clusters view, you will see a list of all your currently provisioned clusters, as well as any clusters that have been deleted within the last four hours. 
+
+For each cluster, you will see the cluster name, type (shared, public or private), cluster ID, provider, region, type, KurrentDB version, status, and date of creation, as well as a set of icons for performing common actions, and a menu with additional actions.
+
+![Clusters list](./images/clusters-list.png)
+
+### Details tab
+
+When you select a cluster from the Clusters list, you will see the cluster details page, which includes the cluster name, ID, provisioning status, mutual TLS status, and date of creation.
+
+![Cluster details - Details](images/cluster-details-tab.png)
+### Infrastructure tab
+
+The Infrastructure tab contains details about the cloud provider, region, type (shared), cluster topology (single node), disk size, and any provider specific details.
+
+![Cluster details - Infrastructure](images/cluster-infra-tab.png)
+
+### Database tab
+
+The Database tab contains details about version of KurrentDB running on the cluster and the health status. The `Version` field shows the major release version of KurrentDB, and `Tag` shows the specific patch version.
+
+![Cluster details - Database](images/cluster-database-tab.png)
+
+### Security tab
+
+The Security tab contains important information related to connecting to your cluster securely.
+
+![Cluster details - Security](images/cluster-security-tab.png)
+
+#### IP Access List
+
+Contains information related to the `IP Access List` that is attached to the cluster. A link is present that allows you to edit the list entries.
+
+#### Admin Credentials
+
+During the deployment process, Kurrent Cloud will automatically generate an initial password for the `admin` user. This password can be unmasked by clicking the button that looks like an eye. The password may also be copied using the button next the to field.
+
+![Cluster details - Security](images/cluster-security-ap-tab.png)
```

Comment:
I would add an explanation of what the "Clear initial admin credentials" button does.